src/osm_configurator/view/toplevelframes/reduction_frame.py

```diff
@@ -257,7 +281,7 @@ def freeze(self):
 
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
```

Comment:
hippedy hoppedy, intractable si not part of the codes property!

Reply:
ok